src/MiNET/MiNET/Blocks/BlockFactory.cs

```diff
@@ -90,7 +90,7 @@ static BlockFactory()
 			for (int i = 0; i < palletSize; i++)
 			{
 				if (BlockPalette[i].Data > 15) continue; // TODO: figure out why palette contains blocks with meta more than 15
-				if (BlockPalette[i].Data == -1) continue; // These are blockstates that does not have a metadata mapping
+				if (BlockPalette[i].Data == -1) BlockPalette[i].Data = 0; // These are blockstates that does not have a metadata mapping (use only dafault value)
```

Comment:
I understand what you mean, but this should never happen. All the blocks that need metadata, should have metadata. Do you have a case when this was missing or?